tests/msc2946_test.go

```diff
@@ -168,7 +169,7 @@ func TestClientSpacesSummary(t *testing.T) {
 	// - Rooms are returned correctly along with the custom fields `room_type`.
 	// - Events are returned correctly.
 	t.Run("query whole graph", func(t *testing.T) {
-		res := alice.MustDo(t, "POST", []string{"_matrix", "client", "unstable", "org.matrix.msc2946", "rooms", root, "spaces"}, map[string]interface{}{})
+		res := alice.MustDo(t, "GET", []string{"_matrix", "client", "unstable", "org.matrix.msc2946", "rooms", root, "spaces"}, nil)
```

Comment:
I think this makes it send `null` as the GET body, which I'm not sure is much better :/

Reply:
I think it causes it to not be sent at all, but I'm not 100% sure on that.

Reply:
I think d36faaaa9f16d3f93e46a4bed8a6c95a0b256768 fixes this.